wp-content/plugins/wporg-learn/views/block-workshop-details.php

```diff
@@ -60,6 +61,7 @@
 				<b><?php esc_html_e( 'Print View', 'wporg-learn' ); ?></b>
 				<span>
 						<a class="components-button is-secondary is-small" href="#" onclick="window.print();">
```

Comment:
Would an `aria-label` on the `a` tag work here? So we could keep the translatable string as one unit ("Print View"), rather than splitting it?

Reply:
Done.